models/intel/machine-translation-nar-ru-en-0001/accuracy-check.yml

```
@@ -0,0 +1,38 @@
+models:
+  - name: machine-translation-nar-ru-en-0001
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: narnmt
+          vocabulary_file: tokenizer_tgt/vocab.json
+          merges_file: tokenizer_tgt/merges.txt
+          output_name: ArgMax_1407/Squeeze
```

Model has no 'ArgMax_1407/Squeeze' output
```suggestion
          output_name: pred/Squeeze
```

---

done